src/main/java/com/mixpanel/android/mpmetrics/MPConfig.java

```diff
@@ -208,7 +212,16 @@ public int getBulkUploadLimit() {
 
     // Target max milliseconds between flushes. This is advisory.
     public int getFlushInterval() {
-        return mFlushInterval;
+        return getFlushInterval(null);
```

Comment:
java will do this optimization for you so that you have literally no performance difference